<launch>
	<node name="rosbag_recorder" pkg="rosbag_recorder" type="rosbag_recorder_server.py" output="screen">
		<remap from="record_topics" to="rosbag_recorder/record_topics" />
		<remap from="stop_recording" to="rosbag_recorder/stop_recording" />
	</node>
</launch>